src/common/CurrencyEngine.ts

```diff
@@ -84,7 +84,6 @@ export class CurrencyEngine<
   currentSettings: any
   timers: any
   walletId: string
-  io: EdgeIo
```

Comment:
How can this be removed? A separate commit for this one would have made it clearer perhaps?

Reply:
It is no longer used. The only property anybody ever accessed was `fetch`, and we don't do that anymore.